@kiva/kv-components/vue/KvPopper.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+	<transition :name="transitionType">
+		<div
+			v-show="show"
+			class="tw-absolute"
+			:style="styles"
+			:aria-hidden="show ? 'false' : 'true'"
+		>
+			<slot></slot>
+		</div>
+	</transition>
+</template>
+
+<script>
+import {
+	onBodyTouchstart,
+	offBodyTouchstart,
+	isTargetElement,
+} from '../utils/touchEvents';
+
+export default {
+	name: 'KvPopper',
+	props: {
+		controller: {
+			validator(value) {
+				if (typeof value === 'string') return true;
+				if (typeof window === 'object'
+					&& 'HTMLElement' in window
+					&& value instanceof HTMLElement) return true;
+				return false;
+			},
+			required: true,
+		},
+		openDelay: { type: Number, default: 0 },
+		closeDelay: { type: Number, default: 200 },
+		// must be defined in our globa/transitions.scss
+		transitionType: { type: String, default: '' },
+		popperPlacement: { type: String, default: 'bottom-start' },
+		popperModifiers: { type: Object, default: () => {} },
+	},
+	data() {
+		return {
+			popper: null,
+			popperPromise: null,
+			styles: {},
+			show: false,
+			timeout: null,
+		};
+	},
+	computed: {
+		reference() {
+			return typeof this.controller === 'string' ? document.getElementById(this.controller) : this.controller;
+		},
+	},
+	watch: {
+		show(showing) {
+			if (this.reference) {
+				this.reference.setAttribute('aria-expanded', showing ? 'true' : 'false');
+			}
+			this.$emit(showing ? 'show' : 'hide');
+		},
+	},
+	mounted() {
+		this.reference.tabIndex = 0;
+		this.attachEvents();
+	},
+	updated() {
+		if (this.popper) {
+			this.popper.scheduleUpdate();
+		}
+	},
+	beforeDestroy() {
+		this.removeEvents();
+		if (this.popper) {
+			this.popper.destroy();
+		}
+	},
+	methods: {
+		open() {
+			this.initPopper().then(() => {
+				this.setTimeout(() => {
+					this.show = true;
+					this.attachBodyEvents();
```

Comment:
Right above this line, please add `this.popper.scheduleUpdate();` (see here: https://github.com/kiva/ui/pull/5324/commits/a3155cacde7e298b7a81dd814d5f1ed4b04ef616). This was a change needed for Vue 3 to ensure the tooltip was in the right location.

Reply:
Thanks for the heads up.